### PR TITLE
cleanup: minimize diff with 1.9.10

### PR DESCRIPTION
**Pull request by smartcontracts**
## Description
Attempting to reduce the overall diff as much as possible. For the sake of legibility & portability.

Plz refer to this link to see the diff: https://github.com/ethereum-optimism/go-ethereum/compare/v1.9.10...cleanup/minimize-diff

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](./CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -164,15 +166,17 @@ var (
...
 		EnvVar: "NETWORK_ID",
 	}
 	ChainIdFlag = cli.Uint64Flag{
-		Name:   "chainid",
-		Usage:  "Chain ID identifier",
-		Value:  420,
+		Name:  "chainid",
+		Usage: "Chain ID identifier",
+		Value: 420,
+
```
Comment (tynes) [inline cmd/utils/flags.go]: No need for whitespace here as this is a completely new option

File: core/vm/interpreter.go [hunk 2]
```diff
@@ -280,8 +276,7 @@ func (in *EVMInterpreter) Run(contract *Contract, input []byte, readOnly bool) (
 		// if the operation clears the return data (e.g. it has returning data)
 		// set the last return to the result of the operation.
 		if operation.returns {
-			in.returnData = res
-			returnDataCopy = string(res)
+			in.returnData = common.CopyBytes(res)
```
Comment (tynes) [inline core/vm/interpreter.go:279]: Good looks

File: cmd/utils/flags.go (hunk 1 above)
Comment (smartcontracts) [inline cmd/utils/flags.go]:
```suggestion
```

File: internal/ethapi/api.go [hunk 3]
```diff
@@ -536,8 +536,7 @@ func (s *PublicBlockChainAPI) GetBalance(ctx context.Context, address common.Add
 	if state == nil || err != nil {
 		return nil, err
 	}
-	balance := state.GetOVMBalance(address)
-	return (*hexutil.Big)(balance), state.Error()
+	return (*hexutil.Big)(state.GetOVMBalance(address)), state.Error()
```
Comment (tynes) [inline internal/ethapi/api.go:539]: I suppose one line of diff is better than 2 lines of diff but we start to have a lot going on on a single like when we combine things like this

File: internal/ethapi/api.go [hunk 4]
```diff
@@ -656,19 +649,9 @@ func (s *PublicBlockChainAPI) GetProof(ctx context.Context, address common.Addre
 			if storageError != nil {
 				return nil, storageError
 			}
-			// by default, the GetProof API does not return if a storage item
-			// was mutated or not.
```
Comment (tynes) [inline internal/ethapi/api.go:660]: I think this was more readable which should be what we optimize for with new features

File: internal/ethapi/api.go [hunk 5]
```diff
@@ -607,11 +605,6 @@ func (s *PublicBlockChainAPI) GetStateDiffProof(ctx context.Context, blockNrOrHa
 			return nil, err
 		}
 
-		// iterate over all the proofs and set their mutated bit
-		for i := range res.StorageProof {
```
Comment (tynes) [inline internal/ethapi/api.go:611]: This is no longer needed?

**Comment by tynes** [comment]
Will rebase and sync to make sure nothing is broken, it looks good to me besides the comments above

File: internal/ethapi/api.go (hunk 5 above)
Comment (tynes) [reply internal/ethapi/api.go:611]: Confirmed that this is not needed

**Review by tynes — APPROVED** [review]

**Pull request merged**